public/js/main.js

```diff
@@ -141,6 +142,10 @@ function processKeyPoints(combinedKeyPoints) {
 async function startEngine(canvas, video) {
   (function update() {
     computeCombinedKeyPoints(video)
+      .then((combinedFeatures) => {
+        const noHandFound = !combinedFeatures[1].length;
+        return noHandFound ? [combinedFeatures[0], null] : combinedFeatures;
```

Comment:
this feels like it should happen in computeCombinedKeyPoints no?

Reply:
Yeah, great point. I'll move that there.